src/main/java/ca/uhn/fhir/jpa/starter/dotbase/accesslog/AccessLogProvider.java

```diff
@@ -11,6 +11,8 @@
 import javax.servlet.http.HttpServletRequest;
 import org.hl7.fhir.r4.model.Basic;
 import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.DateTimeType;
+import org.hl7.fhir.r4.model.Reference;
 import org.hl7.fhir.r4.model.Bundle.BundleEntryComponent;
```

Comment:
```suggestion
import org.hl7.fhir.r4.model.Bundle.BundleEntryComponent;
import org.hl7.fhir.r4.model.Bundle.BundleType;
```

As suggested below, Bundle normally should have a mandatory attribute type. 

Could you add it for accessLog as well in line 54?  I forgot to add a type there too.